Remove Reg_Info (#135)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

Protected_Header = {
  1   => int             ; algorithm identifier,
  4   => bstr            ; Key ID (kid),

>   13  => CWT_Claims      ; CBOR Web Token Claims,

Author: ```suggestion
  15  => CWT_Claims      ; CBOR Web Token Claims,
```

### Registration Policy Metadata

SCITT payloads are opaque to Transparency Services.

> For interoperability, Registration Policy decisions should be based on interpretation of the mandatory metadata in the non-opaque Envelope of a Signed Statement.

Reviewer: ```suggestion
For interoperability, Registration Policy decisions should be based on interpretation of the mandatory metadata in the protected header of a Signed Statement.
```

1. **Signature verification:** The Transparency Service MUST verify the signature of the Signed Statement, as described in {{RFC9360}}, using the signature algorithm and verification key of the Issuer.
1. **Signed Statement validation:** The Transparency Service MUST check that the Signed Statement includes the required protected headers listed above.
The Transparency Service MAY verify the Statement payload format, content and other optional properties.

> 1. **Apply Registration Policy:** For named policies, the Transparency Service MUST check the attributes required by the policy are present in the protected headers.

Reviewer 2: What does a named policy mean in this context?

> 1. **Apply Registration Policy:** For named policies, the Transparency Service MUST check the attributes required by the policy are present in the protected headers.

Reviewer: ```suggestion
1. **Apply Registration Policy:** The Transparency Service MUST check the attributes required by the policy are present in the protected headers.
```

  * tstr => any
}

Protected_Header = {
  1   => int             ; algorithm identifier,
  4   => bstr            ; Key ID,

>   13  => CWT_Claims      ; CBOR Web Token Claims,

Author: ```suggestion
  15  => CWT_Claims      ; CBOR Web Token Claims,
```

- The SCITT version header MUST be included and its value match the `version` field of the Receipt structure
- The DID of Issuer header (in Signed Statements) MUST be included and its value match the `ts_identifier` field of the Receipt structure

> - Transparency Service MUST include the Registration policy info header to indicate to Verifiers what policies have been applied at the registration of this Statement

Reviewer: ```suggestion
- Transparency Service MAY include the Registration policy info header to indicate to Verifiers what policies have been applied at the registration of this Statement
```

I don't think we should force transparency services to disclose such information.

Reviewer: also with the CDDL removed, this cannot be a MUST.
